docs/examples/tutorial/string/for_bytes.pyx

```diff
@@ -1,6 +1,6 @@
 cdef bytes bytes_string = b"hello to A bytes' world"
 
-cdef char c
+cdef Py_UCS4 c
```

Comment:
This doesn't look right to me

Reply:
update to keep c as char, but casting the c in the for loop to byte.